tests/publiccloud/transfer_repos.pm

```diff
@@ -39,6 +39,14 @@ sub run {
 
         # We need to exclude embargoed incidents
         my @all_repos = split(/,/, get_var('MAINT_TEST_REPO'));
+        if (my @excluded_repos = split(/,/, get_var('EXLUDED_TEST_REPO'))) {
```

Comment:
I believe the `if` is redundant here since the next `for` will not iterate on an empty list.

Reply:
eh, its worse ... I `get_var` returns `undef` its throws nice traceback into log ( but still works )